include/seqan3/alignment/configuration/align_config_mode.hpp

```diff
@@ -17,7 +17,11 @@
 #include <seqan3/alignment/configuration/detail.hpp>
 #include <seqan3/core/algorithm/pipeable_config_element.hpp>
 
-namespace seqan3::detail
+namespace seqan3
+{
+
+// We need to nest the namespace explicitly, such that doxygen is capable of correct parsing.
```

Comment:
I'd remove the comment from here since it will affect every nested namespace.

Reply:
Fine, then I remove the comments.